src/components/Layout/DrawerAppBar/DrawerAppBar.js

```diff
@@ -12,27 +12,72 @@ import MenuIcon from '@mui/icons-material/Menu';
 import Toolbar from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
-import {Outlet, useNavigate} from "react-router-dom";
 import { styled, alpha } from '@mui/material/styles';
 import SearchIcon from '@mui/icons-material/Search';
 import {InputBase} from "@mui/material";
 import ButtonBase from "@mui/material/ButtonBase";
-
+import {Outlet, useNavigate} from "react-router-dom";
+import {useDispatch, useSelector} from "react-redux";
+import {logout} from "store/auth/authActions";
+import {ADMIN, GUEST, USER} from "roles/Roles";
 import classes from './DrawerAppBar.module.scss';
 
 const drawerWidth = 240;
-const navItems = [
-    {name: 'Login', path: '/login'},
-    {name: 'Register', path: '/register'},
-    {name: 'Books', path: '/books'},
-    {name: 'Authors', path: '/authors'},
-    {name: 'Profile', path: '/users/id'},
-    {name: 'My Books', path: '/collections/id'},
-    // {name: 'New Book', path: '/newBook'},
-    // {name: 'New Author', path: '/newAuthor'},
-    // {name: 'Change Password', path: '/changePassword'}
+
+const navigationItems = [
+    {
+        label: 'Login',
+        path: '/login',
+        role: [GUEST]
+    },
+    {
+        label: 'Register',
+        path: '/register',
+        role: [GUEST]
+    },
+    {
+        label: 'Books',
+        path: '/books',
+        role: [GUEST, USER, ADMIN]
+    },
+    {
+        label: 'Authors',
+        path: '/authors',
+        role: [GUEST, USER, ADMIN]
+    },
+    {
+        label: 'Profile',
+        path: '/users/id',
+        role: [USER]
+    },
+    {
+        label: 'My books',
+        path: '/collections/id',
+        role: [USER]
+    },
+    {
+        label: 'New Book',
+        path: '/newBook',
+        role: [ADMIN]
+    },
+    {
+        label: 'New Author',
+        path: '/newAuthor',
+        role: [ADMIN]
+    },
+    {
+        label: 'Change Password',
+        path: '/changePassword',
+        role: [ADMIN, USER]
```

Comment:
this route doesn't really belong in the app bar, feels a bit out of place. We will probably put in in the Profile page, let's remove it from the app bar